cmd/hologram-agent/main.go

```diff
@@ -62,10 +64,18 @@ func main() {
 		log.Debug("Using command-line remote address.")
 		config.Host = *dialAddress
 	}
+	if *accessKey != "" {
```

Comment:
Interesting.

My first thought is "don't run as root!". In that case, I'd propose creating a hologram user at which point this problem would still exist.

I then started thinking about providing a path in the hologram config to your aws/config file but that also seems a little weird.

I think from a simplicity standpoint, creating a symlink from ~/.aws to /root/.aws isn't that big of a deal for the user and greatly simplifies your code.

Definitely open to other ideas here.